### DISCUSSION TITLE
Enable collaborator to Download Original Quality Video

### DISCUSSION THREAD

**Discussion by bobbyrabbit** [elonen/clapshot#31] [Ideas]
This is an excellent system.
Once the the collaborator has created their comments and the edits have been completed and re-uploaded for the collaborator to check and sign off. Is there a way to download the original quality video rather than the transcoded version? At the moment the only option I have to download the video is by right-clicking and save as?

**Comment by elonen** [comment]
There's currently no GUI for it, but the URL is predictable (just replace `video.mp4` with `orig/<original-filename>`), so this will be very easy to implement.

E.g. copy-paste the original file name and change URL `/videos/90d1e656/video.mp4` to `/videos/orig/90d1e656/orig/NASA_Red_Lettuce_excerpt.mov`
The browser will then download the original.

Added issue #32 to track the idea.

**Comment by elonen** [comment]
Version 0.6.0 now has an icon at the top of the video player for this.
